independent-publisher-connectors/ClickUp Team Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1231 @@
+            "type": "string",
+			"x-ms-summary":"Create folder new folder",
+			"x-ms-url-encoding": "single",
+			"description":"Create a new folder in your space"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id"
+					  
+                },
+                "name": {
+                  "type": "string",
+                  "description": "Name of folder being created",
+				  "x-ms-summary":"Folder Name"
+					  
+                },
+                "orderindex": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Task order index",
+				  "x-ms-summary":"Order Index"
+					  
+                },
+                "override_statuses": {
+                  "type": "boolean",
+                  "description": "override_statuses",
+				  "x-ms-summary":"Override Statuses"
+					  
+                },
+                "hidden": {
+                  "type": "boolean",
+                  "description": "hidden",
+				  "x-ms-summary":"Hidden Folder"
+					  
+                },
+                "space": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "id",
+					  "x-ms-summary":"Space ID",
+					  "x-ms-visibility":"internal"
+					  
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name",
+					  "x-ms-summary":"Space Name",
+					  "x-ms-visibility":"internal"
+					  
+                    },
+                    "access": {
+                      "type": "boolean",
+                      "description": "access",
+					  "x-ms-visibility":"internal"
+					  
+                    }
+                  },
+                  "description": "space"
+                },
+                "task_count": {
+                  "type": "string",
+                  "description": "task_count",
+				  "x-ms-summary":"Task Count"
+					  
+                },
+                "archived": {
+                  "type": "boolean",
+                  "description": "archived"
+					  
+                },
+                "statuses": {
+                  "type": "array",
+                  "items": {},
+                  "description": "statuses",
+				  "x-ms-summary":"Statuses"
+					  
+                },
+                "lists": {
+                  "type": "array",
+                  "items": {},
+                  "description": "lists",
+				  "x-ms-visibility":"internal"
+					  
+                },
+                "permission_level": {
+                  "type": "string",
+                  "description": "permission_level",

```

Comment:
@marsurice All the description values are expected to be free from grammatical and spell errors.